packages/radix-ui-themes/src/components/base-button.css

```diff
@@ -310,6 +306,10 @@
 .rt-BaseButton:where(.rt-variant-soft) {
   background-color: var(--accent-a3);
 
+  &:where(:focus-visible) {
+    outline: 2px solid var(--accent-8);
+    outline-offset: -1px;
+  }
```

Comment:
Ended up picking the outline color for soft variant on case by case basis, as theme accent still worked in a couple cases, like for soft Switch and Slider